src/itn_api/api.py

```diff
@@ -19,6 +19,7 @@
 from contextlib import asynccontextmanager
 from pathlib import Path
 from typing import Final
+import folium
```

Comment:
It looks like this dependency needs to be added in the requirements.txt for the repo to work.